monitor/xfeemngr/xfeemngr.go

```diff
@@ -27,8 +27,20 @@ type Manager struct {
 	ticker  ticker.Ticker
 }
 
-// feeOracleSyncInterval is the interval at which fee oracles syncs buffered gas and token prices with FeeOracle deployments.
-const feeOracleSyncInterval = 5 * time.Minute
+const (
+	// feeOracleSyncInterval is the interval at which fee oracles syncs buffered gas and token prices with FeeOracle deployments.
+	feeOracleSyncInterval = 5 * time.Minute
+
+	// tokenPriceBufferThreshold is the pct threshold at which a new token price is buffered.
+	tokenPriceBufferThreshold = 0.1
+
+	// gasPriceBufferThreshold is the pct threshold at which a new gas price is buffered.
+	gasPriceBufferThreshold = 0.1
+
+	// GasPriceShield is the pct offset above the buffered gas price the oracle sets on chain
+	// Setting shield == buffer threshold ensures that on chain gas price is always at least as high as the live gas price.
+	GasPriceShield = 0.1
```

Comment:
Should we call this `GasPriceBuffer` instead?

Reply:
I wish, it would be more appropriate, but we already have something called the gas price buffer